src/main/java/mate/academy/spring/validation/PasswordValidator.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.spring.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.dto.request.UserRequestDto;
+
+public class PasswordValidator implements ConstraintValidator<Password, UserRequestDto> {
+    @Override
+    public boolean isValid(UserRequestDto userRequestDto,
+                           ConstraintValidatorContext constraintValidatorContext) {
+        return userRequestDto.getPassword().equals(userRequestDto.getRepeatPassword());

```

Comment:
```suggestion
        return Object.equals(userRequestDto.getPassword(), userRequestDto.getRepeatPassword());

```
This is how it looks like